.github/pull_request_template.md

```diff
@@ -0,0 +1,39 @@
+<!--- Until this PR is ready for review, you can include [WIP] in the title, or create a draft PR. -->
+
+
+<!---
+Below is a suggested pull request template. Feel free to add more details you feel are relevant/necessary.
+
+-->
+
+<!-- 
+Please indicate after the # which issue you're closing with this PR, if applicable.
+If the PR closes multiple issues, include "Fix" before each one is listed.
+You can also link to other issues if necessary, e.g. "See also #1234".
+
+https://help.github.com/articles/closing-issues-using-keywords
+-->
+- Fix #
```

Comment:
It’s a personal preference (also a bit nitpicky) but I think “Closes” is better than “Fix” (?) since not all the issues are bug reports. 

Also if this is changed then Line 11 should be changed to match

Reply:
I see it as fixing the issue, but I'm fine with either 😅